### PR TITLE
change api from coinmarketcap to coingecko

### PR DESCRIPTION
**Pull request by ykris45**
for help to fix issues #250

### CODE REVIEW

File: bot/modules/stats.js [hunk 1]
```diff
@@ -33,18 +33,24 @@ exports.stats = {
...
+            needle.get('https://api.coingecko.com/api/v3/coins/markets?vs_currency=eur&ids=lbry-credits&order=market_cap_desc&per_page=100&page=1&sparkline=false', function(error, response) {
               if (error || response.statusCode !== 200) {
                 msg.channel.send('coinmarketcap API is not available');
               } else {
                 data = response.body.data;
-                let price_eur = Number(data.quotes.EUR.price);
+                let price_eur = Number(data.current_price);
+                needle.get('https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=lbry-credits&order=market_cap_desc&per_page=100&page=1&sparkline=false', function(error, response) {
+              if (error || response.statusCode !== 200) {
+                msg.channel.send('coinmarketcap API is not available');
```
Comment (nikooo777) [inline bot/modules/stats.js]: this should say coingecko, same for a few others above

Comment (nikooo777) [reply bot/modules/stats.js]: also the variable name is `price_eur` but it's usd

Comment (ykris45) [reply bot/modules/stats.js]: @nikooo777 resolved on https://github.com/lbryio/lbry-wunderbot/pull/266/commits/5f825b9156594b49f1eefdf57e29138d898c2b2c

**Comment by ykris45** [comment]
Thanks for merge this PR @Coolguy3289

**Pull request merged**